validator/keymanager/remote-web3signer/v1/custom_mappers.go

```diff
@@ -0,0 +1,399 @@
+	}, nil
+}
+
+// MapDeposit maps the eth2.Deposit proto to the Web3Signer spec.
+func MapDeposit(deposit *ethpb.Deposit) (*Deposit, error) {
+	if deposit == nil {
+		return nil, fmt.Errorf("deposit is nil")
+	}
+	proof := make([]string, len(deposit.Proof))
+	for i, p := range deposit.Proof {
+		proof[i] = hexutil.Encode(p)
+	}
+	return &Deposit{
+		Proof: proof,
+		Data: &DepositData{
+			PublicKey: hexutil.Encode(
+				deposit.Data.PublicKey,
+			),
+			WithdrawalCredentials: hexutil.Encode(
+				deposit.Data.WithdrawalCredentials,
+			),
+			Amount: fmt.Sprint(deposit.Data.Amount),
+			Signature: hexutil.Encode(
+				deposit.Data.Signature,
+			),
+		},
+	}, nil
+}
+
+// MapSignedVoluntaryExit maps the eth2.SignedVoluntaryExit proto to the Web3Signer spec.
+func MapSignedVoluntaryExit(signedVoluntaryExit *ethpb.SignedVoluntaryExit) (*SignedVoluntaryExit, error) {
+	if signedVoluntaryExit == nil {
+		return nil, fmt.Errorf("signed voluntary exit is nil")
+	}
+	return &SignedVoluntaryExit{
+		Message: &VoluntaryExit{
+			Epoch:          fmt.Sprint(signedVoluntaryExit.Exit.Epoch),
+			ValidatorIndex: fmt.Sprint(signedVoluntaryExit.Exit.ValidatorIndex),
+		},
+		Signature: hexutil.Encode(
+			signedVoluntaryExit.Signature,
+		),
+	}, nil
+}
+
+// MapBeaconBlockAltair maps the eth2.BeaconBlockAltair proto to the Web3Signer spec.
+func MapBeaconBlockAltair(block *ethpb.BeaconBlockAltair) (*BeaconBlockAltair, error) {
+	if block == nil {
+		return nil, fmt.Errorf("beacon block altair is nil")
+	}
+	body, err := MapBeaconBlockBodyAltair(block.Body)
+	if err != nil {
+		return nil, errors.Wrap(err, "could not map beacon block body for altair")
+	}
+	return &BeaconBlockAltair{
+		Slot: fmt.Sprint(block.Slot),
+		Body: body,
+		ParentRoot: hexutil.Encode(
+			block.ParentRoot,
+		),
+		StateRoot: hexutil.Encode(
+			block.StateRoot,
+		),
+	}, nil
+}
+
+// MapBeaconBlockBodyAltair maps the eth2.BeaconBlockBodyAltair proto to the Web3Signer spec.
+func MapBeaconBlockBodyAltair(body *ethpb.BeaconBlockBodyAltair) (*BeaconBlockBodyAltair, error) {
+	if body == nil {
+		return nil, fmt.Errorf("beacon block body altair is nil")
+	}
+
+	block := &BeaconBlockBodyAltair{
+		RandaoReveal: hexutil.Encode(body.RandaoReveal),
+		Eth1Data: &Eth1Data{
+			DepositRoot:  hexutil.Encode(body.Eth1Data.DepositRoot),
+			DepositCount: fmt.Sprint(body.Eth1Data.DepositCount),
+			BlockHash:    hexutil.Encode(body.Eth1Data.BlockHash),
+		},
+		Graffiti:          hexutil.Encode(body.Graffiti),
+		ProposerSlashings: make([]*ProposerSlashing, len(body.ProposerSlashings)),
+		AttesterSlashings: make([]*AttesterSlashing, len(body.AttesterSlashings)),
+		Attestations:      make([]*Attestation, len(body.Attestations)),
+		Deposits:          make([]*Deposit, len(body.Deposits)),
+		VoluntaryExits:    make([]*SignedVoluntaryExit, len(body.VoluntaryExits)),
+		SyncAggregate: &SyncAggregate{
+			SyncCommitteeBits:      hexutil.Encode(body.SyncAggregate.SyncCommitteeBits),
```

Comment:
I think SyncCommitteeBits being a special type, (Bitvector), not sure if the hexutil will directly represent it as we need. You might want to call the .Bytes() method of bitvectors and bitlists https://github.com/prysmaticlabs/go-bitfield/blob/385d8c5e3fb7f24c83e25f32dd9511f5462ea0b8/bitvector512.go#L73 before doing hexutil.Encode